examples/knative/powershell/kn-ps-vsphere-inv-slack/test/test-payload-2.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
Is that file used somewhere?

Reply:
No, its reference in README as an additional payload example